presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcResultSet.java

```diff
@@ -116,13 +116,6 @@ public void testPrimitiveTypes()
         checkRepresentation("cast('foo' as char(5))", Types.CHAR, "foo  ");
```

Comment:
Can you explain what is the ordering rule and motivation in the commit message?

Reply:
I just wanted to have simple types before composite types.
And standard types before extensions/"less standard" types.
I do not want to make a rule out of that though. (Too many
rules are not helpful, even if every each of them is sound.)